addon/components/labeled-radio-button.js

```diff
@@ -24,7 +24,11 @@ export default Component.extend({
 
   actions: {
     innerRadioChanged(value) {
-      this.sendAction('changed', value);
+      if (!this.changed || typeof this.changed !== 'function') {
```

Comment:
this commit should be a separate PR, and it should come with a major version bump and updates to the readme

Reply:
that major version bump can be combined with dropping ember support before the current LTS.

Reply:
I've removed the commit but this version is still only compatible with Ember CLI 2.18+

I've updated the readme to include 2 two versions of the install instructions, one for < 2.18 and one for 2.18+

I'll submit another PR with the changes to use closure actions after any issues with this PR have been resolved